src/core/util/animation/animateCss.js

```diff
@@ -315,16 +315,18 @@ if (angular.version.minor >= 4) {
 
       function parseMaxTime(str) {
         var maxValue = 0;
-        var values = (str || "").split(/\s*,\s*/);
-        forEach(values, function(value) {
-          // it's always safe to consider only second values and omit `ms` values since
-          // getComputedStyle will always handle the conversion for us
-          if (value.charAt(value.length - 1) == 's') {
-            value = value.substring(0, value.length - 1);
-          }
-          value = parseFloat(value) || 0;
-          maxValue = maxValue ? Math.max(value, maxValue) : value;
-        });
+        if (str) {
```

Comment:
Can you explain how this `if` statement actually fixes the issue (ideally with a test)?

The following `(str || "")` should catch the case of str being undefined/null and make the code work. What is the value of `str` that is causing this to fail?